resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -2861,6 +2861,764 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
           <hashTree/>
         </hashTree>
       </hashTree>
+      <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Identity Info Schema Type (Execution)" enabled="true">
```

Comment:
Identity Info Schema Type API - Response Assertion - why error is not asserted for [] ?

Reply:
The correct response for this API consist of **"errors":null** rather than "errors":[]. So that is why have kept the response assertion like this.